node/src/blockchain/blockchain_bridge.rs

```diff
@@ -653,13 +455,171 @@ mod tests {
             }],
         };
 
-        let result = subject.handle_report_accounts_payable(request);
+        let result = subject.handle_report_accounts_payable_inner(request);
 
-        assert_eq!(result.0, Err("No consuming wallet specified".to_string()));
+        assert_eq!(result, Err("No consuming wallet specified".to_string()));
+    }
+
+    #[test]
+    fn report_accounts_payable_sends_transactions_to_blockchain_interface() {
```

Comment:
My bad. This should rather tell:  handle_report_accounts_payable_transacts_and_sends_finished_payments_back_to_accountant()